app/helpers/buyers/accounts_helper.rb

```diff
@@ -15,7 +15,7 @@ def link_to_buyer_or_deleted( buyer, path_method = :admin_buyers_account_path)
     if buyer
       if can? :manage, :partners
         path = path_method.is_a?(Symbol) ? send( path_method, buyer) : path_method
-        link_to buyer.org_name, path, :title => account_title(buyer)
+        link_to buyer.org_name, path
```

Comment:
This is kind of a "breaking change", but keeping this requires including the details of the account's user admin (and hence another query to the DB), and I think this is pretty useless, the only thing it does is appending the name of the first admin user to the name of the account, on hovering over the name.

![Screenshot from 2023-05-11 14-38-15](https://github.com/3scale/porta/assets/1270328/8c593f3f-2652-40f0-bc06-5c20510e1bd1)

I don't find it too useful.

Reply:
I reverted this change... After eager loading the admin user the effect it has on the page load time is not noticable.

Reply:
You may check an example of an N+1 test in [test/integration/admin/api/application_plan_limits_controller_test.rb](https://github.com/3scale/porta/blob/0dfe792afb9d5caa31c325dfd86b6b54314a1d25/test/integration/admin/api/application_plan_limits_controller_test.rb#L39-L62).